### PR TITLE
[hi] Added new glossary definition files, from tag storage to tag workload

### PR DESCRIPTION
**Pull request by utkarsh-singh1**
Description:

Localised glossary contents in Hindi.

fixes https://github.com/kubernetes/website/issues/44962

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
Related Page -> https://kubernetes.io/hi/docs/reference/glossary/?storage=true&tool=true&user-type=true&workload=true

File: content/hi/docs/reference/glossary/code-contributor.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Code Contributor
```
Comment (sftim) [inline content/hi/docs/reference/glossary/code-contributor.md:2]: I'd expect to see Hindi words here.

File: content/hi/docs/reference/glossary/developer.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Developer (disambiguation)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/developer.md:2]: I'd expect to see Hindi words here.

File: content/hi/docs/reference/glossary/dynamic-volume-provisioning.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Dynamic Volume Provisioning
```
Comment (sftim) [inline content/hi/docs/reference/glossary/dynamic-volume-provisioning.md:2]: I'd expect to see Hindi words here.

File: content/hi/docs/reference/glossary/persistent-volume-claim.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Persistent Volume Claim
```
Comment (sftim) [inline content/hi/docs/reference/glossary/persistent-volume-claim.md:2]: Better to write PersistentVolumeClaim. The words “persistent”, ”volume“ and “claim” don't appear in Hindi.

File: content/hi/docs/reference/glossary/platform-developer.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+title: Platform Developer
```
Comment (sftim) [inline content/hi/docs/reference/glossary/platform-developer.md:2]: I'd expect to see Hindi words here.

File: content/hi/docs/reference/glossary/probe.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: Probe
```
Comment (sftim) [inline content/hi/docs/reference/glossary/probe.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/replica.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+---
+title: Replica
```
Comment (sftim) [inline content/hi/docs/reference/glossary/replica.md:2]: I'd expect to see Hindi  here; you can use `aka` to also show the English term.

File: content/hi/docs/reference/glossary/storage-class.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+title: Storage Class
```
Comment (sftim) [inline content/hi/docs/reference/glossary/storage-class.md:2]: I'd write StorageClass; the words “storage” and “class” don't appear in Hindi.

File: content/hi/docs/reference/glossary/persistent-volume.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+---
+title: Persistent Volume
```
Comment (sftim) [inline content/hi/docs/reference/glossary/persistent-volume.md:2]: Better to write PersistentVolume. The words “persistent” and ”volume“ don't appear in Hindi.

File: content/hi/docs/reference/glossary/cluster-architect.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+---
+title: Cluster Architect
```
Comment (sftim) [inline content/hi/docs/reference/glossary/cluster-architect.md:2]: I'd expect to see Hindi words here.

**Comment by utkarsh-singh1** [comment]
@sftim , Updated the PR, PTAL.

**Comment by utkarsh-singh1** [comment]
@sftim @divya-mohan0209, Is there any changed needed in this PR or is it ready for merge.

**Pull request closed without merging**